rpmlint/configdefaults.toml

```diff
@@ -288,6 +289,12 @@ ValidLicenseExceptions = []
 # Default white list for PAM modules
 PAMModulesWhiteList = []
 
+# Default list of essential dependencies for SLP policy check
+SLPEssentialDependencies = []
+
+# Default list of legacy exceptions for SLP policy check
+SLPPolicyLegacyExceptions = []
```

Comment:
No exeptions.

Reply:
I agree with this. Otherwise this check makes no sense.